tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "compilerOptions": {
+    "allowJs": true,
+    "esModuleInterop": true,
+    "exactOptionalPropertyTypes": true,
+    "forceConsistentCasingInFileNames": true,
+    "inlineSources": true,
```

Comment:
We could migrate the sourcemap settings to the build tsconfig here as well

Reply:
Also we should probably exclude `dist`

Reply:
Agree. Fixed in [f045643](https://github.com/MetaMask/eth-json-rpc-infura/pull/58/commits/f04564357b90a56f7c371e7b881246abb37fee44).